style(OutlinedInput):  update disabled input text color (#2339)
toptal/picasso: .changeset/swift-badgers-appear.md

---
'@toptal/picasso': patch
---

> Disabled form inputs in Picasso should have text in the appropriate color: grey.main2 (#84888E)

Reviewer: ```suggestion
Fix of text color of disabled form input. The new color is grey.main2 (#84888E).
```
